triage/experiments/validate.py

```diff
@@ -0,0 +1,221 @@
+import logging
+import architect
+from architect.utils import convert_str_to_relativedelta
+from triage.validation_primitives import table_should_have_data,\
+    column_should_be_intlike,\
+    column_should_be_booleanlike,\
+    column_should_be_stringlike,\
+    column_should_be_timelike
+
+
+class Validator(object):
+    def __init__(self, db_engine):
+        self.db_engine = db_engine
+
+
+class FeatureAggregationsValidator(Validator):
```

Comment:
Is most of this just moving from [architect.FeatureGenerator](https://github.com/dssg/architect/blob/master/architect/feature_generators.py#L168)? If so, we should probably include the imputation validation here, too (and probably a reasonable time to go ahead and address [this issue](https://github.com/dssg/architect/issues/47) while we're at it). Also, is there an architect PR to remove from there as well?

Reply:
Yep, lots of things in here are just moving from Architect. It's easy enough to move the imputation stuff from Architect.

The collate issue seems like it will be less needed post-merge. 

Removing things from architect I figured we would do after this PR. Though now, with the code merge, I'm not sure if it's worth it. Maybe architect could keep its validations as it rides off into the sunset.

Reply:
Makes sense on holding off on removing things from architect, though I think the collate issue will still be important even post-monolith in terms of making things a little bit more clear and easier to edit so you only have to do so in one place in the code (even if the monolith will at least mean you only have to edit one repo regardless).